ui/src/adapters/api/credentials.ts

```diff
@@ -426,10 +624,12 @@ const authQRCodeParser = getStrictParser<AuthQRCodeInput, AuthQRCode>()(
 
 export async function createAuthQRCode({
   env,
+  issuerIdentifier,
```

Comment:
We need to rename "QRCode" endpoints, because they are not generating qr codes. Maybe "QRCode" could be substituted with  "Message". CC: @demonsh